lib/index.d.ts

```diff
@@ -0,0 +1,17 @@
+export interface highlightOptions {
+  html?: boolean,
+  classPrefix?: string,
+  colors?: {
+    keyword: string,
+    function: string,
+    number: string,
+    string: string,
+    special: string,
+    bracket: string,
+    clear: string
+  }
+}
+
+declare module 'sql-highlight' {
+  export function highlight(sqlString: string, options?:highlightOptions): string;
+}
```

Comment:
Formatting according to my other TypeScript projects as well as capitalizing `highlightOptions`.

```suggestion
export interface HighlightOptions {
  html?: boolean;
  classPrefix?: string;
  colors?: {
    keyword: string;
    function: string;
    number: string;
    string: string;
    special: string;
    bracket: string;
    clear: string;
  };
}

declare module 'sql-highlight' {
  export function highlight(sqlString: string, options?: HighlightOptions): string
}

```